doc/man3/OSSL_PARAM_int.pod

```diff
@@ -295,7 +272,7 @@ This example is for setting parameters on some object:
     const char *foo = "some string";
     size_t foo_l = strlen(foo) + 1;
     const char bar[] = "some other string";
-    const OSSL_PARAM set[] = {
+    OSSL_PARAM set[] = {
```

Comment:
Unnecessary removal of const

Reply:
I've changed it back, hopefully people won't be confused :)